src/model/enum/LayerType.ts

```diff
@@ -1,3 +1,3 @@
-export type LayerType = 'TILEWMS' | 'VECTORTILE' | 'WFS' | 'WMS' | 'WMTS' | 'XYZ' | 'WMSTIME' | 'MAPBOXSTYLE';
+export type LayerType = 'TILEWMS' | 'VECTORTILE' | 'WFS' | 'WMS' | 'WMTS' | 'XYZ' | 'WMSTIME' | 'MAPBOXSTYLE' | 'MVT';
```

Comment:
Can we also add the type to the SHOGun [backend enum](https://github.com/terrestris/shogun/blob/main/shogun-lib/src/main/java/de/terrestris/shogun/lib/enumeration/LayerType.java)?

And not related to this PR in particular, but do we actually need the `'VECTORTILE'` type? Shouldn't we make this as specific as possible like `MVT`?

Reply:
Exactly, generic `VECTORTILE` type can be removed since there doesn't exist a specific parser. See fa27d2c549a7a20fced9ca838b5069fe468ea7c4

Reply:
Added to `shogun` in https://github.com/terrestris/shogun/pull/990